Resources/Prototypes/_LostParadise/Entities/Objects/Consumable/Drinks/drinks_bottles.yml

```
@@ -0,0 +1,247 @@
+  - type: SolutionContainerVisuals
+    maxFillLevels: 6
+    fillBaseName: fill-
+  - type: Sealable
+
+- type: entity
+  parent: [DrinkBottleVisualsOpenable, DrinkBottlePlasticBaseFull]
+  id: LPPDrinkOrangeTeaBottleFull
+  name: orange tea bottle
+  description: Bottle of orange tea with citrus freshness.
+  components:
+  - type: Item
```

yamllint

[error] wrong indentation: expected 4 but found 2

(indentation)